pages/_app.tsx

```diff
@@ -24,11 +25,10 @@ interface MyAppProps extends AppProps {
 }
 
 function MyApp(props: MyAppProps) {
-  const [cookies] = useCookies(['darkMode', 'timestamp', 'analytics']);
   const { Component, pageProps } = props;
   const router = useRouter();
   useEffect(() => {
-    if (gtag.GA_TRACKING_ID && cookies.analytics === 'true') {
+    if (gtag.GA_TRACKING_ID) {
```

Comment:
`&& cookies.analytics === 'true'` is required here as well as in `_document.tsx` in order to prevent tracking analytics or get errors

Reply:
done